packages/lib/src/vis/shared/AxisSystem.tsx

```diff
@@ -13,25 +14,31 @@ interface Props {
   title?: string;
 }
 
+const v1 = new Vector3();
+const v2 = new Vector3();
+
 function AxisSystem(props: Props) {
   const { axisOffsets, title } = props;
 
   const { abscissaConfig, ordinateConfig, abscissaScale, ordinateScale } =
     useAxisSystemContext();
 
-  const { position, zoom } = useThree((state) => state.camera);
+  const camera = useThree((state) => state.camera);
   const canvasSize = useThree((state) => state.size);
   const { width, height } = canvasSize;
 
+  const unprojectedLeftBottom = v1.set(-1, -1, 0).unproject(camera);
```

Comment:
`unproject` allows to move from the normalized camera space to the world space. 

The nice thing about the normalized space is that the FOV is always delimited by `(-1, -1)` and `(1,1)` which means I don't have to care about the `canvasSize` anymore: all is handled by `unproject`.

Reply:
Oh, and I need to set `v1` at each computation as `unproject` mutates it...

Reply:
This is awesome. Instead of mutating the vector, you could clone it. Here is what I'm thinking, with some naming suggestions:

```ts
const NORM_BOTTOM_LEFT= new Vector(-1, -1, 0);
...
const worldBottomLeft = NORM_BOTTOM_LEFT.clone().unproject(camera);
```